pandas/tests/indexes/multi/test_contains.py

```diff
@@ -58,6 +58,25 @@ def test_isin():
     assert result.dtype == np.bool_
 
 
+@pytest.mark.parametrize(['values', 'badix'],
```

Comment:
we tend to use the string style i.e. 'values, badix' and probably expected_badix would be clearer

Reply:
Done